Classes/Controller/BackendServiceController.php

```diff
@@ -348,12 +355,16 @@ public function changeBaseWorkspaceAction(string $targetWorkspaceName, NodeInter
     /**
      * Persists the clipboard node on copy
      *
-     * @param NodeInterface $node
+     * @param array $nodes
      * @return void
      */
-    public function copyNodeAction(NodeInterface $node)
+    public function copyNodesAction(array $nodes)
     {
-        $this->clipboard->copyNode($node);
+        // TODO @christianm want's to have a property mapper for this
```

Comment:
Is this Todo just a relict as you use the propertyMapper?

Reply:
Oh no, @kitsunet said that in the future he wants to have all of this stuff to happen automatically on the property mapper level, so the comment should stay to keep reminding us of it :)